### ISSUE TITLE
tox-gh-actions prevents an explicitly specified TOXENV from being run

### ISSUE THREAD

**Issue by brechtm** [ymyzk/tox-gh-actions#34]
When explicitly specifying a tox environment to run by setting the TOXENV environment variable, tox-gh-actions will not run any tox environment. It doesn't matter whether the environment is one that is specified in tox.ini or not.

```
$ GITHUB_ACTION=1 TOXENV=regression tox -v
using tox.ini: /Users/brechtm/Documents/Code/rinohtype/tox.ini (pid 13403)
using tox-3.20.0 from /Users/brechtm/Documents/Code/rinohtype/.venv/lib/python3.8/site-packages/tox/__init__.py (pid 13403)
_________________________________________ summary _________________________________________
  congratulations :)
```

**Comment by brechtm** [comment]
I think maybe #24 would fix this.

**Comment by ymyzk** [comment]
@brechtm, thanks for opening a ticket. This is actually by design at this point and it seems not consistent with toe-travis. What tox-gh-actions is trying to do is finding a list of environment from `["regression"]` which satisfy the following factor constraints:
```
[gh-actions]
python =
    2.7: py27
    3.6: py36
    3.7: py37
    3.8: py38
    3.9: py39
    pypy3: pypy3
```
As a result, tox-gh-actions is ignoring all environments. To avoid this problem, we can update the `tox.ini` like this assuming that we want to run regression tests on Python 3.8:
```
[gh-actions]
python =
    2.7: py27
    3.6: py36
    3.7: py37
    3.8: py38, regression
    3.9: py39
    pypy3: pypy3
```

**Comment by ymyzk** [comment]
Okay, it seems tox-travis is allowing to override a list of environments by setting `TOXENV` or passing `-e` option. I think it's fine to implement a similar behavior for tox-travis. As it's a breaking change, I'll estimate impact a bit more.

**Comment by brechtm** [comment]
> ```
> [gh-actions]
> python =
>     2.7: py27
>     3.6: py36
>     3.7: py37
>     3.8: py38, regression
>     3.9: py39
>     pypy3: pypy3
>```

This would run

**Comment by ymyzk** [comment]
This feature is shipped in v2.0.0
